docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -33,11 +33,117 @@ Understanding GBD's Disease and Health Categories
 GBD Definitions
 ---------------
 
-- Include the GBD definitions for each of these different categories 
+These definitions and others can be found on the `glossary Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=INTRANET&title=IHME+Glossary>`_. 
 
-Understanding the GBD Hierarchy
--------------------------------
+.. list-table:: GBD definitions of terms 
+  :widths: 15 15 15
+  :header-rows: 1
 
-- Showing what is a direct subset, what overlaps, what interacts but is independent 
-- Include some information on modeling implications as a result of this 
+  * - Term 
+    - Definition
```

Comment:
Oh, I now see that all of these definitions come from the IHME Glossary. I guess I don't find their definitions super useful, but maybe it's good to keep them for consistency and then just add extra context/etc. to the notes section.